code/game/objects/effects/spawners/f13lootdrop.dm

```diff
@@ -818,7 +818,7 @@
 	name = "very low tier non-hobo ballistic gun"
 	loot = list(/obj/effect/spawner/bundle/f13/ninemil,
 				/obj/effect/spawner/bundle/f13/caravan_shotgun,
-				/obj/effect/spawner/bundle/f13/mosin,
+				//obj/effect/spawner/bundle/f13/mosin, No. More. Mosin.
```

Comment:
Dont need to remove it from the actual loot altogether. If that's no good, you can rename and reflavor the mosin to some kind of generic hunting rifle.

Reply:
Generic hunting rifle was already in the game. I've replaced some references to mosin with the varmint rifle, others with the hunting rifle.